src/store/setToolCursor.js

```diff
@@ -0,0 +1,53 @@
+import { state, modules } from './index.js';
+
+export { setToolCursor, resetToolCursor, hideToolCursor };
+
+const globalConfiguration = modules.globalConfiguration;
+
+/**
+ * Creates an SVG Cursor for the target element
+ *
+ * @param {MouseCursor} svgCursor - The cursor.
+ */
+function setToolCursor(element, svgCursor) {
+  if (!globalConfiguration.state.showSVGCursors) {
+    return;
+  }
+  console.log('attempting to set cursor...');
```

Comment:
We really should look into a logging library where we can set log levels in `init`

Reply:
Yeah that'd be a good idea.